app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,84 @@
+class FriendshipsController < ApplicationController
+  before_action :set_friendship, only: %i[show]
+
+  # GET /friendships
+  # GET /friendships.json
+  def index
+    @friendships = Friendship.all
+  end
+
+  # GET /friendships/1
+  # GET /friendships/1.json
+  def show; end
+
+  # GET /friendships/new
+  def new
+    @friendship = Friendship.new
+  end
+
+  # GET /friendships/1/edit
+  def edit; end
+
+  # POST /friendships
+  # POST /friendships.json
+  def create
+    @friendship = Friendship.new(sender_id: params[:sender], receiver_id: params[:receiver])
+
+    if @friendship.save
+      flash[:notice] = 'Friendship was successfully created.'
+    else
+      flash[:alert] = 'Friendship not created!'
+    end
+    redirect_to user_path(params[:receiver])
+  end
+
+  def destroy_both
+    @friendship = Friendship.where(sender_id: params[:receiver], receiver_id: params[:sender]).first
+    if @friendship.destroy
+      puts('>>>>>>>>>>>>>>>>>>First')
+      destroy
+    else
+      flash[:alert] = 'Friendship not removed!'
+      redirect_to user_path(params[:receiver])
+    end
+  end
+
+  # PATCH/PUT /friendships/1
+  # PATCH/PUT /friendships/1.json
+  def update
+    @friendship = Friendship.where(sender_id: params[:sender], receiver_id: params[:receiver]).first
+
+    if @friendship.update(status: params[:status])
+      flash[:notice] = 'Friendship was successfully ' + params[:status] + '.'
+    else
+      flash[:alert] = 'Friendship not ' + params[:status] + '!'
+    end
+    redirect_to user_path(params[:sender])
+  end
+
+  # DELETE /friendships/1
+  # DELETE /friendships/1.json
+  def destroy
+    @friendship = Friendship.where(sender_id: params[:sender], receiver_id: params[:receiver]).first
+
+    if @friendship.destroy
+      puts('>>>>>>>>>>>>>>>>>>Second')
+      flash[:notice] = 'Friendship was successfully removed.'
+    else
```

Comment:
## Remove puts statement
- Please remove the inline comments code, because it can cause security issues when your application starts growing